PatchesAndAreasApi.Tests/V1/E2ETests/Steps/ReplacePatchResponsibleEntitiesStep.cs

```diff
@@ -0,0 +1,95 @@
+using FluentAssertions;
+using Hackney.Shared.PatchesAndAreas.Boundary.Request;
+using Hackney.Shared.PatchesAndAreas.Domain;
+using Hackney.Shared.PatchesAndAreas.Infrastructure;
+using Newtonsoft.Json;
+using PatchesAndAreasApi.Tests.V1.E2ETests.Fixtures;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System;
+using Hackney.Shared.PatchesAndAreas.Infrastructure.Constants;
+using System.Collections.Generic;
+
+namespace PatchesAndAreasApi.Tests.V1.E2ETests.Steps
+{
+    public class ReplacePatchResponsibleEntitiesStep : BaseSteps
+    {
+        public ReplacePatchResponsibleEntitiesStep(HttpClient httpClient) : base(httpClient)
+        { }
+
+        /// <summary>
+        /// You can use jwt.io to decode the token - it is the same one we'd use on dev, etc. 
+        /// </summary>
+        /// <param name="requestObject"></param>
+        /// <returns></returns>
+        public async Task<HttpResponseMessage> CallAPI(Guid id, List<ResponsibleEntities> responsibleEntities, int? ifMatch)
+        {
+            var token =
+                "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMTUwMTgxMTYwOTIwOTg2NzYxMTMiLCJlbWFpbCI6ImUyZS10ZXN0aW5nQGRldmVsb3BtZW50LmNvbSIsImlzcyI6IkhhY2tuZXkiLCJuYW1lIjoiVGVzdGVyIiwiZ3JvdXBzIjpbImUyZS10ZXN0aW5nIl0sImlhdCI6MTYyMzA1ODIzMn0.SooWAr-NUZLwW8brgiGpi2jZdWjyZBwp4GJikn0PvEw";
```

Comment:
I think could potentially be a security risk - the E2E token is good for creating and querying data - should it not be expressed in an env var or pipeline var?

Reply:
I don't think there is any security risk - if you try to use this token for any API request then it would return 403 forbidden.